cmd/agent/install_script.sh

```diff
@@ -372,10 +369,10 @@ background and submit metrics to Datadog.
 
 If you ever want to stop the Agent, run:
 
-    $stop_instructions
+    %s
```

Comment:
This is less readable than before. Is there any reason to use printf? With a simple "echo" we wouldn't need to do this. This applies to pretty much every printf in the file.

Reply:
Yeah, since we can probably switch to `echo` here since we don't seem to be using any of the features of `printf`.